Userland/Libraries/LibWeb/DOM/Node.h

```diff
@@ -66,7 +66,7 @@ class Node : public EventTarget {
     bool is_character_data() const { return type() == NodeType::TEXT_NODE || type() == NodeType::COMMENT_NODE; }
     bool is_document_fragment() const { return type() == NodeType::DOCUMENT_FRAGMENT_NODE; }
     bool is_parent_node() const { return is_element() || is_document() || is_document_fragment(); }
-    bool is_slottable() const { return is_element() || is_text(); }
+    bool is_slottable() const { return is_element() || is_text() || is_cdata_section(); }
```

Comment:
Does this mean the cast in `Node::as_slottable()` is incorrect too?

```cpp
Slottable Node::as_slottable()
{
    VERIFY(is_slottable());

    if (is_element())
        return JS::NonnullGCPtr { static_cast<Element&>(*this) };
    return JS::NonnullGCPtr { static_cast<Text&>(*this) };
}
```

Reply:
I think this should be OK because `CDATASection` is a subclass of `Text`. 

It makes me wonder if we should have a separate NodeType for `CDATASection` at all. It seems slightly unintuitive to me that `is_text()` returns false for `CDATASection`, but casting it to `Text` is fine.

Reply:
Ah duh, I thought `CDATASection` / `Text` were the other way around.
Unfortunately the `NodeType` is required by the web platform, so we can't just remove it.

Agree that it's weird that `is_text()` would return false for `CDATASection`, that seems like a bug IMO!